src/assets/accessories/finger_nails/finger_nails.asset.ts

```diff
@@ -0,0 +1,54 @@
+DefineAsset({
+	name: 'Artificial finger nails',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		nails: {
+			name: 'Nails',
+			default: '#AC0909',
+		},
+	},
+	// size:150, y:550, X:0
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Accessory',
```

Comment:
Also should provide the to be added attribute `Hand_item`. See comment in "attributes.ts".